packages/core/src/model.ts

```diff
@@ -240,12 +267,12 @@ export class ChartModel {
 		}) as any;
 	}
 
-	getStackedData({ percentage } = { percentage: false }) {
+	getStackedData(groups?: string[], { percentage } = { percentage: false }) {
```

Comment:
`groups` should be part of the original parameter in the function like `percentage`

Reply:
when the function is called we'd want to see `getStackedData({ groups: blah, percentage: blah });`

rather than `getStackedData(blah, blah)`

Reply:
oh yes, good note - thanks 👍